observation_portal/sciapplications/filters.py

```diff
@@ -20,6 +20,7 @@ class ScienceApplicationFilter(django_filters.FilterSet):
     only_authored = django_filters.BooleanFilter(
         method='filter_only_authored'
     )
+    tag = django_filters.CharFilter(method='filter_has_tag', label='Has tag')
```

Comment:
These might be better as a MultipleChoiceFilter? With the choices defined by the tag set for your account.

Reply:
Hmm, I'm not sure if there's an easy way to get the user account of the current user in a filterset to be able to filter down the apps so I can get the list of available tags for the choices... Will look into it.